### DISCUSSION TITLE
One Radio Recording w/ Song Titles?

### DISCUSSION THREAD

**Discussion by Hi-ImKyle** [AzuraCast/AzuraCast#6638] [Q&A] [answered]
I have a 2 hour 37 minute long radio recording, (_It's Non Stop Pop FM, from GTA 5_), and I want to add all the songs at the timestamps to report back to the player and show the correct song information for the timestamp. But I'm not sure if this feature exists and I don't know what you would call it? 

Basically I'd ideally like to set say 15:31 -> 19:50 to be Daryl Hall and John Oates - Adult Education, because that's what the timestamp is for that song, then say after that like 19:50 -> 20:10 is Intermission, then the next song and so on and so on. 

Is this a thing? Am I missing it? What would you even call such a feature?

**Comment by Piratenkanon** [comment]
Only one tag can be saved in an MP3 file, otherwise the MP3 file must be split

**Reply by Hi-ImKyle** [reply]
Pretty bad given my use case. Maybe a future feature

**Comment by paulcjkp** [comment, marked as answer]
You could do that with cue sheets, but it's not implemented yet.  It's a suggested feature here https://azuracast.featureupvote.com/suggestions/39020/support-cue-sheets

**Reply by Hi-ImKyle** [reply]
Annoying that it's some third party website just to suggest features, but yeah that sounds exactly like what I need
